mmv1/provider/terraform/common~compile.yaml

```diff
@@ -46,3 +46,4 @@
 <% end -%>
 'main.go': 'third_party/terraform/main.go.erb'
 'go.mod': 'third_party/terraform/go.mod.erb'
+'.goreleaser.yml': 'third_party/terraform/.goreleaser.yml'
```

Comment:
We need this line to put the file in the repo root, I'd assume?

Reply:
yes! seemed that way.